torch/_torch_docs.py

```diff
@@ -7995,29 +8009,34 @@ def merge_dicts(*dicts):
            r"""
 tensor(data, *, dtype=None, device=None, requires_grad=False, pin_memory=False) -> Tensor
 
-Constructs a tensor with :attr:`data`.
+Constructs a tensor with no autograd history (a "leaf tensor") by copying :attr:`data`.
 
 .. warning::
 
-    :func:`torch.tensor` always copies :attr:`data`. If you have a Tensor
-    ``data`` and want to avoid a copy, use :func:`torch.Tensor.requires_grad_`
-    or :func:`torch.Tensor.detach`.
-    If you have a NumPy ``ndarray`` and want to avoid a copy, use
-    :func:`torch.as_tensor`.
+    When working with tensors prefer using :func:`torch.Tensor.clone`,
+    :func:`torch.Tensor.detach`, and :func:`torch.Tensor.requires_grad_` for
+    readability. Letting `t` be a tensor, ``torch.tensor(t)`` is equivalent to
+    ``t.clone().detach()``, and ``torch.tensor(t, requires_grad=True)``
+    is equivalent to ``t.clone().detach().requires_grad_(True)``.
+
+    To change whether a tensor requires grad without copying it use just
```

Comment:
This paragraph is not as clear as the others.
Maybe something saying that changing if a Tensor requires grad can be done in other ways:
- inplace via requires_grad_
- as a new Tensor that is a view of the original view detach
- as a new Tensor that is not a view as mentioned above via clone().detach()

Reply:
Good point; I'll try to revise it with your thinking